xbmc/playlists/PlayListXSPF.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright (C) 2018 Tyler Szabo
```

Comment:
Could you change this to:

```
/*
 *  Copyright (C) 2018 Tyler Szabo
 *  Copyright (C) 2018 Team Kodi
 *  This file is part of Kodi - https://kodi.tv
 *
 *  SPDX-License-Identifier: GPL-2.0-or-later
 *  See LICENSES/README.md for more information.
 */
```

Reply:
you didn't follow my instructions ;)

Reply:
I had to look at this 3 times to realize I missed the `This file is part of Kodi - https://kodi.tv` line. Sorry about that.

Reply:
I verified the lines are verbatim in 3 of the 4 files I added. `PlayListXSPF.h` has Team Kodi only.